CMakeLists.txt

```diff
@@ -17,20 +17,21 @@ if (NOT DEFINED TI_VERSION_MAJOR)
     set(TI_VERSION_PATCH ${CMAKE_MATCH_3})
 endif()
 
+set(CMAKE_COLOR_DIAGNOSTICS ON)
 set(CMAKE_CXX_STANDARD 17)
 set(CMAKE_CXX_STANDARD_REQUIRED TRUE)
-set(CMAKE_EXPORT_COMPILECOMMANDS ON)
+set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
```

Comment:
What does this do? Looks like it was broken before? What was the impact of it being broken? What is the benefit of fixing it?